Makefile.include

```diff
@@ -507,6 +507,13 @@ define check_cmd
 	    exit 1;}
 endef
 
+# Check setsid command is available
+#   This is not the case on MacOSX, so it must be built on the fly
+ifneq (0,$(shell which setsid 2>&1 > /dev/null ; echo $$?))
+  export SETSID = $(RIOTTOOLS)/setsid/setsid
```

Comment:
@cladmi is this the kind of places where target-export-variables should be used?